docker/run_jupyter.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-jupyter notebook --no-browser --allow-root --ip 0.0.0.0
+jupyter notebook --no-browser --ip 0.0.0.0
 
```

Comment:
Could you remove this redundant empty line?

Reply:
fixed, check and resolve